### DISCUSSION TITLE
docker 镜像无法拉取，试过很多次了

### DISCUSSION THREAD

**Discussion by wanna-diary** [ArtalkJS/Artalk#932] [Q&A] [answered]
root@hecs-407061:/home/docker-linux/conf/conf.d# docker pull artalk/artalk-go
Using default tag: latest
latest: Pulling from artalk/artalk-go
d25f557d7f31: Retrying in 1 second 
56cd91f8c558: Retrying in 1 second 
418d1a7835ff: Retrying in 1 second 
4bc83f191a13: Waiting 
723bc3c75ac6: Waiting 
0dc55fa91051: Waiting 
4f4fb700ef54: Waiting 
error pulling image configuration: download failed after attempts=6: dial tcp 66.220.149.18:443: i/o timeout

**Comment by wanna-diary** [comment]
windows、Mac、云服务器（Ubuntu）都试过，没有网络或者代理的问题

**Comment by GoodBoyboy666** [comment, marked as answer]
> windows、Mac、云服务器（Ubuntu）都试过，没有网络或者代理的问题

被墙了，自己去搜一下就知道了

**Reply by Chapliing** [reply]
你好，我在redhat服务器里面pull一个docker镜像的时候，报错是“error pulling image configuration: download failed after attempts=6: dial tcp 114.43.24.59:443: i/o timeout”，换了很多源都没用，请问该怎么解决呢，下面是我的“daemon.json”文件：

{
  "registry-mirrors": [
    "https://docker.mirrors.ustc.edu.cn",
    "https://6kx4zyno.mirror.aliyuncs.com",
    "https://docker.nju.edu.cn"
  ]
}

**Reply by GoodBoyboy666** [reply]
ustc的镜像因为某些不可描述的原因关了，nju不知道关没关，中间那个阿里云的不知道是不是你自己自建的，是的话留一个阿里云就行。从你发的报错来看IP定位到的是台湾，所以应该不是从aliyun拉的镜像。如果中间那个阿里云不是你自己自建的，那也没啥用。目前国内流行的几个docker hub公共镜像站差不多都嘎了，最好的还是自建或者magic

**Reply by dianso** [reply]
> 你好，我在redhat服务器里面pull一个docker镜像的时候，报错是“error pulling image configuration: download failed after attempts=6: dial tcp 114.43.24.59:443: i/o timeout”，换了很多源都没用，请问该怎么解决呢，下面是我的“daemon.json”文件：
> 
> { "registry-mirrors": [ "https://docker.mirrors.ustc.edu.cn", "https://6kx4zyno.mirror.aliyuncs.com", "https://docker.nju.edu.cn" ] }

用我的试试

https://docker.beitai.cc/
